include/defines.h

```diff
@@ -156,12 +156,27 @@
  * the battle courses could be displayed in the same
  * way race courses are.
  */
-#define MUSHROOM_CUP 0
-#define FLOWER_CUP   1
-#define STAR_CUP     2
-#define SPECIAL_CUP  3
-#define BATTLE_CUP   4
-#define NUM_CUPS     5
+enum {
+    MUSHROOM_CUP,
+    FLOWER_CUP,
+    STAR_CUP,
+    SPECIAL_CUP,
+    BATTLE_CUP,
+    NUM_CUPS
+};
+
+#define NUM_COURSES_PER_CUP 4
+
+/**
+ * @brief Options for gCourseIndexInCup
+ */
+enum {
+    FIRST_COURSE_IN_CUP,
+    SECOND_COURSE_IN_CUP,
+    THIRD_COURSE_IN_CUP,
+    FOURTH_COURSE_IN_CUP
+};
```

Comment:
Could we keep `CUP_COURSE_ONE` etc. ?

Reply:
for me FIRST_COURSE_IN_CUP are more clear then CUP_COURSE_ONE because I understand CUP_COURSE_ONE as the first CUP

Reply:
It doesn't really fit the naming convention though

Reply:
if you want we can remove the IN key word but it's just to be more precise of what it is

Reply:
just replace by COURSE_ONE, COURSE_TWO, COURSE_THREE, COURSE_FOUR so just remove the prefix CUP_

Reply:
That's fine